### ISSUE TITLE
Unknown burst type: 1101 (mototurbo from rtl dongle)

### ISSUE THREAD

**Issue by antonjan** [szechyjs/dsd#58]
When decoding mototurbo on Linux.  I normally get the following error on the first two lines of decoding.
1) Linux   uname -a
Linux anton-Laptop-sdr-610 3.8.0-33-generic #48~precise1-Ubuntu SMP Thu Oct 24 16:31:16 UTC 2013 i686 i686 i386 GNU/Linux

2) DSD 1.7.0-dev
3) RTL v0.5.3
4) sudo rtl_fm -d0 -f 438.225M -p39 -s 48K | sudo dsd -fr -i - -o pa:1

Error from dsd
Sync:  +DMR       mod: QPSK inlvl: 10%  slot0  [slot1]  Unknown burst type: 1101
Sync:  +DMR       mod: C4FM inlvl:  9% [slot0]  slot1   Unknown burst type: 1101
Sync:  +DMR       mod: GFSK inlvl:  8%  slot0  [slot1]  Slot idle
Sync:  +DMR       mod: GFSK inlvl:  8% [slot0]  slot1   RATE 3/4 DATA
Sync:  +DMR       mod: GFSK inlvl:  9%  slot0  [slot1]  RATE 3/4 DATA
Sync:  +DMR       mod: GFSK inlvl:  9% [slot0]  slot1   RATE 3/4 DATA
Sync:  +DMR       mod: GFSK inlvl:  8%  slot0  [slot1]  Slot idle
Sync:  +DMR       mod: GFSK inlvl:  8% [slot0]  slot1   Slot idle
Sync:  +DMR       mod: GFSK inlvl:  8%  slot0  [slot1]  Slot idle

![image](https://cloud.githubusercontent.com/assets/2240283/8532134/6e9714d6-242c-11e5-9f65-9ed13466368a.png)

Can get audio from websdr http://zr6aic.giga.co.za:8902   frequency at 438.225Mhz (DMR sdr)

**Comment by szechyjs** [comment]
The latest DMR standard `ETSI TS 102 361-1 V2.2.1 (2013-02)` has the following data types labeled "Reserved for future use".

```
1011
1100
1101
1110
1111
```

**Comment by antonjan** [comment]
According to my dock it has the following details

1011 Group data activity on BS time slot 1
1100 Emergency group voice activity on BS time slot 1
1101 Emergency individual voice activity on BS time slot 1
1110 Reserved
1111 Reserved

hope this help

**Comment by szechyjs** [comment]
@antonjan you are looking at the wrong part of the protocol. Refer to the following sections of `ETSI TS 102 361-1 V2.2.1 (2013-02)` describing the Frame SYNChronization

- 6.2
- 9.1.3
- 9.3.6

**Comment by twodogs** [comment]
I've just confirmed in v2.3.1 of the standard, and it's still labelled  as "Reserved for future use", yet 95% of all DMR traffic in my area now comes up this way.   There just isn't a Linux option that I can find that lets me listen to Amateur DMR here in the US.

**Comment by francisuk1989** [comment]
> The latest DMR standard `ETSI TS 102 361-1 V2.2.1 (2013-02)` has the following data types labeled "Reserved for future use".
> 
> ```
> 1011
> 1100
> 1101
> 1110
> 1111
> ```

any update to theses feature labeled "Reserved for future use"

They are most used in cap+ systems, 1111,1101

[2022-05-28 23:41:16] Sync:  +DMR       mod: C4FM inlvl: 26%  slot1  [slot2] | Color Code=05 (CRC ERR) | RATE 1/2 DATA
[2022-05-28 23:41:16] Sync:  +DMR       mod: C4FM inlvl: 25%  slot1  [slot2] | Color Code=05 (CRC ERR) | Unknown burst type: 1111
[2022-05-28 23:41:16] Sync:  +DMR       mod: GFSK inlvl: 28%  slot1  [slot2] | Color Code=05 (OK)      | Slot idle    
[2022-05-28 23:40:04] Sync:  +DMR       mod: GFSK inlvl: 23%  slot1  [slot2] | Color Code=05 (CRC ERR) | Unknown burst type: 1101

**Comment by JaySmithWpg** [comment]
I don't think this issue is with DSD itself, I think it's an issue with rtl_fm's output. DMR requires more bandwidth and deviation than standard narrow FM.

I used to have "Unknown Burst Type" problems when using DSD with Gqrx. The default Narrow FM settings resulted in garbage being fed into DSD, which it interpreted as unknown burst types. I resolved the issue by setting "Filter Width" to 12.5k, "Max Dev" (under Narrow FM Mode Options) to 17kHz, and Tau to 0.

Unfortunately I don't have a SDR that's compatible with rtl_fm, so I can't test this theory personally. But I can say that it's working great now that I have Gqrx configured properly.
